.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,46 @@
+name: Bug report
+description: File a bug report to help us improve
+title: "[BUG]"
+labels: ["bug", "🔍 needs-triage"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Contributing directly to this project is always encouraged. More details can be found in the [Contributing Guide](https://github.com/Azure/template-analyzer/blob/development/CONTRIBUTING.md).
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: Please provide a description of the issue you are seeing. If applicable, please include any error messages, exception stacktraces, or memory dumps.
+      placeholder: Description
+    validations:
+      required: true 
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: Provide a description of the expected behavior.
+      placeholder: Expected behavior
+    validations:
+      required: true
+  - type: textarea
+    id: repro-steps
+    attributes:
+      label: Reproduction Steps
+      description: |
+        Please include minimal steps to reproduce the problem if possible. E.g.: the smallest possible code snippet; or a small project, with steps to run it. If possible include text as text rather than screenshots (so it shows up in searches).
+      placeholder: Minimal Reproduction
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: |
+        Please provide more information on your environment:
+          * Hosting platform or OS and Template Analyzer Version (`dotnet --info` output for .NET Core projects)
```

Comment:
I would expect the part in parentheses to clarify what it comes after, but `dotnet --info` doesn't relate to Template Analyzer version.  Can you expand this a bit?  It might help to explain how to get the Template Analyzer version too.  Also, the "for .NET Core projects" piece probably isn't necessary, since we only have our single project.

Reply:
I guess I wasn't super clear here. 😅  Added request for dotnet version too.